pyccl/tests/test_tkkssc.py

```diff
@@ -249,3 +249,146 @@ def test_tkkssc_counterterms_gc(kwargs):
 
     assert np.abs((tk_nogc_12 - tkc12) / tk_gc_12 - 1).max() < 1e-5
     assert np.abs((tk_nogc_34 - tkc34) / tk_gc_34 - 1).max() < 1e-5
+
+
+@pytest.mark.parametrize('kwargs', [{'is_number_counts1': True,
```

Comment:
Can you check if this would work, just to make it a bit less verbose?
`[{f'is_number_counts{i}': nc[i] for i in range(4)} for nc in itertools.product([True,False], repeat=4)]`
you'll need to `import itertools`

Reply:
Nice feature! Done.